guidance.md

```diff
@@ -32,41 +32,43 @@ The request context **SHALL** include:
 ## CDS Service returns a PAMA Response
 
 Since a CDS Services might offer multi-purpose advice, accepting PAMA Requests as well as unrelated requests, the CDS Service must first determine
-whether each incoming draft ServiceRequest is "PAMA-relevant". At a minimum, the CDS Service MUST recognize ServiceRequests that pertain to priority clinical areas defined
+whether each incoming draft ServiceRequest is "PAMA-relevant". At a minimum, the CDS Service SHALL recognize ServiceRequests that pertain to priority clinical areas defined
 by CMS, and MAY recognize additional clinical areas.
 
-For each PAMA-relevant ServiceRequest, the CDS Service MUST provide a PAMA Response by including the required PAMA extensions described below.
+For each PAMA-relevant ServiceRequest, the CDS Service SHALL evaluate its PAMA AUCs and attempt to assign one of three PAMA Rating values (`appropriate` if there is an AUC consistent with the proposed imaging order, `not-appropriate` if there is an AUC that contra-indicates the proposed imaging order, or `not-applicable` if no AUC pertains to the proposed imaging order). In the event that not enough information is available to reach one of these ratings (e.g., if key data have not been captured in a structured form), the CDS Service SHALL NOT produce a PAMA Response for the ServiceRequest, and SHOULD instead offer Cards with app links or suggestions that would lead to a computable rating (e.g., a link to an app that would collect additional information necessary to evaluate an AUC).
```

Comment:
Hey @jmandel , in this paragraph, our third code is `not-applicable`, below, in the table and example, it's `no-criteria-apply`. I'd slightly prefer the `no-criteria-apply` (because it's closer to CMS and HL7v2's terminology), but I believe that they have the same meaning and that the most important thing is that we're consistent!